sys/include/net/netopt.h

```diff
@@ -199,6 +199,14 @@ typedef enum {
      */
     NETOPT_CCA_THRESHOLD,
 
+    /**
+     * @brief get statistics about sent and received packets of the layer
```

Comment:
Hm, With "layer" I would understand something like physical layer, network layer, application layer. UDP is not a layer, "transport layer" would be. But netopts get() would be sent to UDP, only returning UDP's stats, right? Not the stats of the whole "transport layer", e.g., including TCP?